apps/ai-backoffice/app/files/page.tsx

```
@@ -0,0 +1,130 @@
+
+                if (error) {
+                  toast({
+                    variant: 'destructive',
+                    description: 'Failed to download file. Please try again.',
+                  })
+                  return
+                }
+
+                window.location.href = data.signedUrl
+              }}
+            >
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)